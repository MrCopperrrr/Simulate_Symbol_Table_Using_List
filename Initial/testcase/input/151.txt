INSERT s string
ASSIGN s 'abc_1'