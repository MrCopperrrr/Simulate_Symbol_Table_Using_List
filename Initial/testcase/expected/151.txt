Invalid: ASSIGN s 'abc_1'